### DISCUSSION TITLE
@vercel/python file read bug

### DISCUSSION THREAD

**Discussion by jbhoosreddy** [vercel/vercel#4904] [Help] [answered]
Hi folks

I am running into an issue where I am unable to read a file using python's `open` method but I'm able to read it via flasks, `send_file` API.

I have a reproducible example project here: https://github.com/jbhoosreddy/vercel-bug-python-function-file-open

Does anyone have any suggestions how I can work around this issue? It will be very appreciated!

Thanks

**Comment by zvikfir** [comment, marked as answer]
Hey there,

I looked into your problem. It occurs because you use the function readlines(), which returns a list of strings.

You need to return a string. Changing the function to read() solves the problem.
